CHANGELOG.md

```diff
@@ -21,7 +21,12 @@ Enables the use of pre-release builds of Omnisharp. Downloading a pre-release bu
 
 #### Editor
 
-* Updated the Omnisharp Status Bar to appear on left instead of right because it displays the project level info(which is a characteristic of the elements on the left side of the window) and removed ambiguities in its behavior.([#2146](https://github.com/OmniSharp/omnisharp-vscode/issues/2146), PR: [@2133](https://github.com/OmniSharp/omnisharp-vscode/pull/2133))
+* Splits the OmniSharp status bar in two parts both of which appear on the left and have specific responsibilities. ([#2146](https://github.com/OmniSharp/omnisharp-vscode/issues/2146), PR: [@2133](https://github.com/OmniSharp/omnisharp-vscode/pull/2133))
+  * Omnisharp Server Status Bar - Shows the various states that the OmniSharp server initialisation might be in(eg: Downloading, Installing, etc). The flame would be green when the server is initilaised and running properly and red if there is an error.
```

Comment:
* initialisation -> **initialization** _(US English spelling)_
* in(eg: -> **in (eg:** _(need space before open parenthesis)_
* The flame would be green -> The flame **icon is** green _("icon" is the appropriate noun; incorrect tense)_
* initialised -> **initialized** _(US English spelling)_
* properly and red -> **properly, or** red _(insert comma; or instead of and)_